### PR TITLE
Create 09.06.02.config

### PR DESCRIPTION
**Pull request by Tychodewaard**
Add webp support

### CODE REVIEW

File: DNN Platform/Website/Install/Config/09.07.00.config [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<configuration>
+  <nodes configfile="Web.config">
+    <node path="/configuration/system.web/staticContent/ action="update" key="name" collision="overwrite">
```
Comment (bdukes) [inline DNN Platform/Website/Install/Config/09.07.00.config:3]:
```suggestion
    <node path="/configuration/system.web/staticContent/ action="update" key="fileExtension" collision="ignore">
```

**Comment by bdukes** [comment]
Yeah, if we're doing a 9.6.2 release, let's include this.

**Comment by valadas** [comment]
Ok, so let's decide on the next meeting and we can adjust the filename accordingly...

File: DNN Platform/Website/Install/Config/09.06.02.config [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<configuration>
+  <nodes configfile="Web.config">
+    <node path="/configuration/system.web/staticContent/ action="update" key="fileExtension" collision="ignore">
```
Comment (valadas) [inline DNN Platform/Website/Install/Config/09.06.02.config:3]:
```suggestion
    <node path="/configuration/system.web/staticContent/" action="update" key="fileExtension" collision="ignore">
```

There was a missing double-quote that made install/upgrades fail with invalid xml. I will accept my own change here and let it build and try again...

**Pull request merged**